### PR TITLE
feat(stellar): add Stellar ITS deployment script

### PR DESCRIPTION
**Pull request by AttissNgo**
[AXE-6236](https://axelarnetwork.atlassian.net/browse/AXE-6236)
- Add an initial script for deploying ITS contract
- Add an its.js script that allows setting/removing trusted ITS addresses
- Update Stellar README with new commands and edit for readability

[AXE-6236]: https://axelarnetwork.atlassian.net/browse/AXE-6236?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: stellar/README.md [hunk 1]
```diff
@@ -75,6 +75,12 @@ Provide `--estimate-cost` to show the gas costs for the initialize transaction i
 node stellar/deploy-contract.js deploy axelar_operators --wasm-path ../axelar-cgp-soroban/target/wasm32-unknown-unknown/release/axelar_operators.optimized.wasm --initialize
 ```
 
+### Interchain Token Service
+```bash
+node stellar/deploy-contract.js deploy interchain_token_service --wasm-path ../axelar-cgp-soroban/target/wasm32-unknown-unknown/release/interchain_token_service.optimized.wasm --initialize
+```
+
+
```
Comment (milapsheth) [inline stellar/README.md:83]:
```suggestion
```

File: stellar/deploy-contract.js [hunk 2]
```diff
@@ -47,13 +47,25 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
...
+            return { owner };
+
         case 'axelar_operators':
             return { operator };
         default:
             throw new Error(`Unknown contract: ${contractName}`);
     }
 }
 
+function getIntializeFuncName(contractName) {
```
Comment (milapsheth) [inline stellar/deploy-contract.js:60]: Use an object to map between contract name to initialize name maybe?

File: stellar/its.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+async function mainProcessor(processor, args, options) {
+    const config = loadConfig(options.env);
+    const chain = getChainConfig(config, options.chainName);
+
+    const wallet = await getWallet(chain, options);
+
+    if (!chain.contracts?.interchain_token_service) {
+        throw new Error('Interchain Token Service package not found.');
+    }
+
```
Comment (milapsheth) [inline stellar/its.js:35]: add a `prompt(..)` here for consistency (see existing usage)

**Review by milapsheth — APPROVED** [review]

File: stellar/its.js (hunk 3 above)
Comment (AttissNgo) [reply stellar/its.js:35]: I see prompts only on `deploy` and `upgrade` from `deploy-contract.js`. Neither `gateway.js` nor `operator.js` use prompts. What do you want it to say? Just ask the user if they want to continue?

**Pull request merged**